api/src/utils/loadDb.js

```diff
@@ -24,9 +24,59 @@ async function getTaxonomies(model) {
 
 async function getEvents(model) {
   try {
-    let { data } = await axios.get(`${API_URL}/events?client_id=${API_KEY}`);
-    for (let i = 0; i < data.events.length; i++) {
-      const element = data.events[i];
+    let response = []
+    let allResponse = []
+    response = await Promise.all([
```

Comment:
esto podrias hacer un for que vaya pusheando esto a un array y te deje todas las promesas en ese array para hacerle el promise.all y que no quede tan largo el codigo